draft-thomson-httpbis-http2bis.xml

```diff
@@ -3569,6 +3570,37 @@
             the application protocol and cipher suite are independently selected.
           </t>
         </section>
+        <section anchor="tls13">
```

Comment:
Minor editorial note: is it useful to use the `”tls13features”` anchor for consistency with 1.2 and so we don’t have two almost-identical anchors (“TLS13” and “tls13”) in the document?